torch/_inductor/kernel/flex_attention.py

```diff
@@ -679,7 +679,9 @@ class Mode(Enum):
 
 def _get_rocm_config(query, mode: Mode) -> Tuple[int, int, int, int]:
     dtype = query.get_dtype()
-    head_dim = query.get_size()[-1]
+    head_dim = V.graph.sizevars.evaluate_static_shape(
+        sympy.sympify(query.get_size()[-1])
```

Comment:
The sympify here shouldn't be needed, why do you think you need it?

Reply:
This was copy pasta from: https://github.com/pytorch/pytorch/pull/139069